README.md

```diff
@@ -1,42 +1,146 @@
+- Place trainees into well paid jobs (targeting twice the average of non-10 Academy graduates from one's university)
 
 ## Examples of how you probably used this competency today without realizing it
-You're here, and learning - you want to optimize your energy for career success.
 
 ## What components make up this competency?
-- 2 year plan
-- 5 year plan
-- Your website with projects you've done
-- CV
-- 3 references
-- One or more mentors - they could be peers
-- Your GitHub code base
-- A view of employers in the field, and what they are looking to hire
-- A sense of the employment market in your field, in your region and globally
-- Knowledge of the cost of ensuring 24/7 internet access with 95% uptime.
+### Understanding the ways you can be hired
+* Full-time regular (may be remote)
+  * Analogy: Long-term stable relationship.  A commitment is made from both parties to work together and stability provides a place to grow.
+  * Jobs advertised publicly and likely the longest vetting process.
+  * The expectation is that you will be with the company for a period with no fixed end-date in mind.  This allows the company to depend on you and for you to depend on the company.
+  * There is a cancellation notice period from either side - usually 4-12 weeks
+  * This is likely the only possible way for you to get stock options with a company - they don't come without investing something significant into the company, either money or quite a lot of time.
+  * You would likely get benefits from the company.
+* Full-time fixed duration contract (may be remote)
+  * Analogy: Stable relationship while doing a PhD - while you are there, you know that you will make it work, but there is always an end-date in mind.  Stability and growth is there, though very long-term planning may not happen.
+  * Jobs may be advertised publicly, probably a long vetting process.
+  * Both you and the company invest in the other, and continue to extend the contract on a regular basis as needed.
+  * This is a popular model for stable remote work.
+  * You may get some benefits from the company.
+  * You are unlikely to get any stock options.
+* Contract
+  * Analogy: Summer romance
+  * You are hired to deliver a specific task within a certain amount of time.  There is not necessarily an expectation that you are part of regular team meetings and you may be working on items which are outside the regular company workflow.
+  * Jobs may be advertised, but will often accrue through contacts and networks.
+  * Contracts likely in the 3-6 month timeframe.
+  * You are highly unlikely to get any benefits from the company or stock options.
+  * It's very important to make the expectations clear - most contracts will not have this 100% clear at the start, and this is important to clarify - what has to be delivered by when, and when payments are made.
+* Freelance
+  * Upwork etc - not covered here.
+
```

Comment:
Should i add more options here or leave as is

Reply:
If you have more options, please do add.